Synology Readme.md

```diff
@@ -0,0 +1,96 @@
+
+![Download chlabbi/docker-notify docker image](images/1FIleStation.png)
+
+
+### 2. Set your ```config.json``` file.
+This is no different on Synology than [any other platform](https://github.com/Schlabbi/docker-notify).
+
+Place your ```config.json``` file in the folder you defined earlier, ```docker/docker-notify``` in the previous step.
+
+It is a good idea to validate your ```config.json``` using a tool like ```https://jsonlint.com/?code=``` to be sure it is ready before starting. Any error in you config.json may cause the container to continuously restart. You can also use the [example config.json](https://github.com/Schlabbi/docker-notify/blob/master/config.json.example) as a test and once you know the container is working, stop the container, replace the config.json with your own, and then restart the container. 
+
+### 3. Create a ```cache``` folder. 
+In the folder you defined earlier, ```docker/docker-notify``` create a folder called, ```cache```.
+
+
+### 4. Download the Docker image.
+Open the Docker app on your Synology DSM, go to the `Registry` section and then search for and download the [chlabbi/docker-notify](https://github.com/Schlabbi/docker-notify) image.
+
+![Download chlabbi/docker-notify docker image](images/4registry.png)
+
+Make sure you select the `latest` tag when you have the option. 
+
+
+## 5. Create Container.
+When the container download is complete, launch a new container using the [chlabbi/docker-notify](https://github.com/Schlabbi/docker-notify) image.
+
+
+## 6. Name your container.
+Choose a name and then click ```Advanced Settings```:
+
+![Name your container](images/6Create0.png)
+
+
+## 7. Configure Advanced Settings.
+Check the box to ```Enable auto-restart```. This is recommended if you want your container to automatically restart after your Synology DSM reboots or if docker-notify crashes.
+
+![General settings](images/7Create1.png)
+
+
+## 8. Configure the Volume tab.
+Click the ```Volume``` tab and then **Add Folder**.
+
+![Configure Volume info 1](images/8aVolumes.png)
+
+
+Choose the folder— ```docker/docker-notify``` used in this example, and set the **Mount path** to be ```/usr/src/app/cache```.
+
+Next, add **a file** with the mount path as ```/usr/src/app/config.json``` as shown which connects to the config.json file you created previously.
+
+![Configure Volume info 2](images/8bVolumes.png)
+
+## 9. Configure the Network tab.
+Click the ```Network``` tab and check the box to ```Use the same network as Docker host```. 
+
+![Network settings](images/9network.png)
+
+## 10. Configure Environment (optional).
+Click the ```Environment``` tab and add a new environment variable named `checkInterval`. The value of the `checkInterval` defines in hours, how often checks will be made for new container releases.
```

Comment:
I moved that variable into the config file. It was read from the environment, but that was an error because that field existed in the config file schema all the time. Also, this is in minutes.

Reply:
fixed